benchmark/locust/locust_models.py

```
@@ -76,3 +79,108 @@ def validate_host(cls, v: str) -> str:
+        AIPerf runner. Without a sweep this is the base configuration alone,
+        labelled with the empty string.
+        """
+        if not self.sweeps:
+            return [("", self.base_config)]
+
+        # Sorted so the run order and directory names do not depend on YAML key order.
+        keys = sorted(self.sweeps)
+
+        runs = []
+        for combination in product(*(self.sweeps[key] for key in keys)):
+            overrides = dict(zip(keys, combination))
```

Ruff (0.16.1)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)